### ISSUE TITLE
How to specify the version of grpc and protobuf

### ISSUE THREAD

**Issue by xiaolixiaoyi** [rules-proto-grpc/rules_proto_grpc#30]
I used rules_proto_grpc with version 0.2.0 and found that the default version of grpc was 1.22. 
Now I want to update my grpc and protobuf to the latest released version, how can I do this?

**Comment by aaliddell** [comment]
New versions of gRPC have moved some of the plugin locations, so won't work as a direct replacement in the 0.2.0 release. However, you can try the bazel1.0-support branch, which has support for the newer gRPC versions

**Comment by xiaolixiaoyi** [comment]
It works! Thank you!

**Comment by jarivk** [comment]
Is it possible to specify (older) protobuf version when using with rules_proto_grpc (version 0.2.0)?

**Comment by jarivk** [comment]
From [repositories.bzl](https://github.com/rules-proto-grpc/rules_proto_grpc/blob/921654580ac0858a1fd8c60f5f4b9159a2012f0f/repositories.bzl) I see it uses 3.9.0. How can specify 3.6.0?

**Comment by aaliddell** [comment]
Sorry for the slow response @jarivk , I thought this was fixed in 1.0.0 but it's possible the override options don't work for github sourced repos. I'll check this and let you know

**Comment by aaliddell** [comment]
The below should work for selecting an alternate version:

```
rules_proto_grpc_repos(
    com_google_protobuf_ref = "v3.10.0",
    com_google_protobuf_sha256 = "<fill hash here>",
)
```

Although this will only work across dependency versions that are not substantially different from the supported version (e.g. paths the same and same sub-dependencies)
